todoapp/Program.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System;
```

Comment:
Nice job!
item class: 5/5 
dao class: 5/5 
dbContext: 5/5 
utils class: 5/5
app class 10/10
handle bad input: 5/10 
seperation of concerns: 10/10 
print menu: 10/10
add item: 10/10
delete item: 10/10 
mark done: 10/10
show items: 10/10